src/Control/Eff/Internal.hs

```diff
@@ -109,19 +109,17 @@ comp (Arrs f) (Arrs g) = Arrs (f >< g)
 (^|>) :: Arrs r a b -> Arr r b c -> Arrs r a c
 (Arrs f) ^|> g = Arrs (f |> g)
 
--- | The Eff monad (not a transformer!). It is a fairly standard coroutine monad
--- where the type @r@ is the type of effects that can be handled, and the
--- missing type @a@ (from the type application) is the type of value that is
--- returned.  It is NOT a Free monad! There are no Functor constraints.
+-- | The monad that all effectful computation in this library bases on.
```

Comment:
@power-fungus perhaps "The monad that all effects in this library are based on."?